package.yaml

```diff
@@ -1,5 +1,5 @@
 name:    derive-has-field
-version: 0.0.1.5
+version: 0.1.0.0
```

Comment:
IIUC, PVP suggests `MAJOR.MAJOR.MINOR.PATCH`. In this case, I am technically making a backwards incompatible change so I need to bump major version. I probably won't need the first major so I'll leave that zero for now.

Reply:
Side note: historically, I have been doing PVP wrong in this repository :upside_down_face: